src/lib/components/Cytoscape.react.js

```diff
@@ -1,12 +1,16 @@
 /**
- * JavaScript Requirements: cytoscape
+ * JavaScript Requirements: cytoscape, cytoscape-svg
  * React.js requirements: react-cytoscapejs
  */
+import cytoscape from 'cytoscape';
+import svg from 'cytoscape-svg';
```

Comment:
I'm wondering if it's a good idea to add `svg` to the main module, since we want to minimize the file size when loading dash-cytoscape; also, adding one more package here means more dependencies to keep track. This is the reason why we have an [`extra_index.js`](https://github.com/plotly/dash-cytoscape/blob/master/src/lib/extra_index.js) that loads all of the external dependencies. 

Do you believe this should stay in the main file, or would it be better to have it in the `extra_index.js`?

Reply:
That is a good idea, I was not aware of the exta_index.js before.
I just tried to add it to extra index.js and it works as expected :)

However, following that logic I guess it would be a good idea to put the plugins in an new file.
On top it kinda feels wrong to enable the svg export feature with cyto.load_extra_layouts()

Since I do not 100% understand how to implement this I just added a plugins_index.js file as a reminder.

Reply:
Ivo, please add a function directly to `__init__` [similar to this one](https://github.com/IvoLeist/dash-cytoscape/blob/master/dash_cytoscape/__init__.py#L52-L93), except here you would use `_plugins` instead of `_extra`. The name could be `load_plugins`, and an example on how to use it could be added to the `readme` right below [this section](https://github.com/IvoLeist/dash-cytoscape#external-layouts).

Feel free to also load it in your new usage app, if you have time. Thanks!

Reply:
You are totally right on the confusion around extra_index.js. I think it's something I can add in a future release of Cytoscape, inside CONTRIBUTION.md, so people that want to add more plugin could easily do it.

Reply:
Actually, I think this might not be the best way. Maybe it's better to keep it inside `extra_index.js`. More discussions below.